types/block.go

```diff
@@ -1073,12 +1077,17 @@ func (data *Data) Hash() tmbytes.HexBytes {
 }
 
 // ComputeShares splits block data into shares of an original data square and
-// returns them along with an amount of non-redundant shares. The shares
-// returned are padded to complete a square size that is a power of two. The
-// size of the square is computed and recorded in data
-func (data *Data) ComputeShares() (NamespacedShares, int) {
+// returns them along with an amount of non-redundant shares. If a square size
+// of 0 is passed, the shares returned are padded to complete a square size that
+// is a power of two. The size of the square is computed and recorded in data
+func (data *Data) ComputeShares(squareSize uint64) (NamespacedShares, int, error) {
 	// TODO(ismail): splitting into shares should depend on the block size and layout
 	// see: https://github.com/celestiaorg/celestia-specs/blob/master/specs/block_proposer.md#laying-out-transactions-and-messages
```

Comment:
Isn't this already the case by now? Or only after all the abci++ chnages?

Reply:
I'm not entirely sure, as that [linked](https://github.com/celestiaorg/celestia-specs/blob/master/src/specs/block_proposer.md) paragraph is empty, but I think its safe to remove 

https://github.com/celestiaorg/celestia-core/pull/637/commits/760502e590e4bbd4ac97730632c8b2105b0b3cb2